README.md

```diff
@@ -1,20 +1,48 @@
+## Golem
+
+Official Rust implementation of Golem. Golem is a network of nodes that implement the Golem Network protocol. We provide the default implementation of such a node in the form of the Golem daemon, Yagna.
+
+<h5 align="center">
+  <a href='https://golem.network/'><img
+      width='500px'
+      alt=''
+      src="https://user-images.githubusercontent.com/35585644/111472751-939f5100-872a-11eb-8c26-926117080e35.png" /></a>
+  <br/>A flexible, open-source platform for democratised access to digital resources.
+</a>
+</h5>
+
+</p>
+<p align="center">
+    <a href="https://github.com/golemfactory/yagna/workflows/CI/badge.svg" alt="CI">
+        <img src="https://github.com/golemfactory/yagna/workflows/CI/badge.svg" /></a>
+    <a href="https://github.com/golemfactory/yagna/watchers" alt="Watch on GitHub">
+        <img src="https://img.shields.io/github/watchers/golemfactory/yagna.svg?style=social" /></a>
+    <a href="https://github.com/golemfactory/yagna/stargazers" alt="Star on GitHub">
+        <img src="https://img.shields.io/github/stars/golemfactory/yagna.svg?style=social" /></a>
+    <a href="https://discord.gg/y29dtcM" alt="Discord">
+        <img src="https://img.shields.io/discord/684703559954333727?logo=discord" /></a>
+    <a href="https://twitter.com/golemproject" alt="Twitter">
+        <img src="https://img.shields.io/twitter/follow/golemproject?style=social" /></a>
+    <a href="https://reddit.com/GolemProject" alt="Reddit">
+        <img src="https://img.shields.io/reddit/subreddit-subscribers/GolemProject?style=social" /></a>
+</p>
+
+Golem Network has officially gone on mainnet with the [Beta I release](https://blog.golemproject.net/mainnet-release-beta-i/).
+
+Golem democratizes society’s access to computing power by creating a decentralized platform where anyone can build a variety of applications, request computational resources and/or offer their idle systems in exchange for cryptocurrency tokens (GLM). The actors in this decentralized network can assume one of the three non-exclusive roles:
 
 * **Requestor**
 Has a need to use IT resources such as computation hardware. Those resources are purchased in the decentralized market. The actual usage of the resources is backed by Golem's decentralized infrastructure.
 
 * **Provider**
 Has IT resources available that can be shared with other actors in the network. Those resources are sold in the decentralized market.
 
+* **Developer**
+Builds applications to run for requestors on the network. Golem's potential goes much beyond a singular application. In Golem there is not a single monolithic use-case - Golem allows for a self sustainable ecosystem. A myriad of different types of users requesting digital resources, providing them and building applications on top of the platform. See [Awesome Golem](https://github.com/golemfactory/awesome-golem/blob/main/README.md#%EF%B8%8F-apps) for just a taste of the various types of applications that can be built and run on Golem!
```

Comment:
I do not get sense of the below sentence within this context. Can you rephrase? Or enlighten me? :)

> A myriad of different types of users requesting digital resources, providing them and building applications on top of the platform

Reply:
Good point! I've made it more concise.